modules/ftp-client/src/main/java/org/dcache/ftp/client/dc/AbstractDataChannel.java

```diff
@@ -122,7 +122,7 @@ public static void registerHandler(int transferMode,
 	}
 	*/
 
-        logger.debug("registering handler for class " + clazz.toString() + "; id = " + id);
+        logger.debug("registering handler for class {}; id = {}", clazz.toString(), id);
```

Comment:
You are right about not mixing different tasks in one pull request. But in this case, changing `clazz.toString()` to `clazz` is still within the scope of your task. Please go ahead and change it.